index.css

```diff
@@ -112,3 +112,235 @@ ul {
   position: absolute;
   left: 130px;
 }
+
+.work {
+  display: grid;
+  grid-template-columns: none;
+  grid-template-rows: none;
+  justify-content: space-evenly;
+  padding-right: 15px;
+}
+
+#button1 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  width: 90px;
+  background-color: #7d73fc;
+  height: 35px;
+  text-align: center;
+  font-size: 12px;
+  font-family: 'Roboto', sans-serif;
+  color: #edecff;
+}
+
+#button1:hover {
+  cursor: pointer;
+  background-color: purple;
+}
```

Comment:
Good job for using hover 👍 

However can you update the design to stick to Figma
<img width="317" alt="image" src="https://user-images.githubusercontent.com/7243132/170831744-089a5f6b-c8ae-4b3d-b7c1-599a189d5d62.png">